README.md

```diff
@@ -1 +1,54 @@
-# to-do-list
\ No newline at end of file
+# to-do-list
+
+>  To Do List Project For Microverse By Bwalya Nonde.
+
+
+> This is a simple web develoment project for microverse created using simple HTML(Hyper text mark-up   Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage. And JavaScript which is used for making the webpage responsive.
+
+![screenshot](to-do-list.png)
+
+
+## Built With
+
+- html
+- css
+- javascript
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+git clone  
+Main
+cd list-structure
```

Comment:
Kindly edit your `Getting Started` section to give brief steps on how to clone and run the project properly. It's should be something like this:

```
git clone git@github.com:BwalyaNonde/to-do-list.git
cd to-do-list
npm install
npm run build
npm start
```
Then check `http://localhost:8080/`
This is the proper way of cloning the project and running it.

Reply:
@Mre55 Thank you so much for your review and making me a better developer! 👍 

Happy Coding!!